app/views/alchemy/base/500.html.erb

```diff
@@ -12,15 +12,29 @@
   ], search: false
 ) %>
 
+<% content_for(:alchemy_body_class) { 'error' } %>
+
 <%= render_message(:error) do %>
   <h1><%= Alchemy.t('An error happened') %></h1>
   <h2>
     <%= @error.class %>
     <%= @notice %>
   </h2>
-  <p>
+  <p class="hidden">
   <% @trace.each do |line| %>
     <%= line %><br>
   <% end %>
   </p>
+  <a href="#" class="error_details--toggle">
+    <%= Alchemy.t('Show error details') %>
+  </a>
 <% end %>
+
+<script>
+  $(function(){
+    $('.error_details--toggle').click(function(){
+      $('.error.message p').toggleClass('hidden');
+      return false;
+    });
+  });
+</script>
```

Comment:
It's a personal choice, but one could also just do `hide()` and `show()`, as well as `slideDown()` and `slideUp()`, all of which come with small animations.